diff --git a/debian/changelog b/debian/changelog
index 9aedb57..c383f2d 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+main-menu (1.39+SprezzOS) unstable; urgency=low
+
+  * SprezzOS theming.
+
+ -- Nick Black <nick.black@sprezzatech.com>  Sun, 17 Jun 2012 01:11:24 +0000
+
 main-menu (1.38) unstable; urgency=low
 
   * Team upload
diff --git a/debian/main-menu.templates b/debian/main-menu.templates
index 0906dc7..89d4a3f 100644
--- a/debian/main-menu.templates
+++ b/debian/main-menu.templates
@@ -1,7 +1,7 @@
 Template: debian-installer/main-menu-title
 Type: text
 # :sl1:
-_Description: Debian installer main menu
+_Description: SprezzOS installer
 
 Template: debian-installer/main-menu
 Type: select
