diff --git a/debian/netcfg-common.templates b/debian/netcfg-common.templates
index e964a5a..7863eb5 100644
--- a/debian/netcfg-common.templates
+++ b/debian/netcfg-common.templates
@@ -141,7 +141,7 @@ _Description: Failure of key exchange and association
 
 Template: netcfg/get_hostname
 Type: string
-Default: debian
+Default: sprezzabox
 # :sl1:
 _Description: Hostname:
  Please enter the hostname for this system.
diff --git a/debian/changelog b/debian/changelog
index d439743..8bc9c27 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+netcfg (1.92-SprezzOS1) unstable; urgency=low
+
+  * SprezzOS build
+
+ -- Nick Black <nick.black@sprezzatech.com>  Sun, 23 Sep 2012 21:16:34 +0000
+
 netcfg (1.92) unstable; urgency=low
 
   [ Philipp Kern ]
diff --git a/netcfg-common.c b/netcfg-common.c
index ac23be0..088c567 100644
--- a/netcfg-common.c
+++ b/netcfg-common.c
@@ -1038,7 +1038,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char *host
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             *hostname = '\0';
         }
 
@@ -1063,7 +1063,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char *host
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             *hostname = '\0';
         } else {
             break;
