diff --git a/debian/changelog b/debian/changelog
index b4c231a..8ab1d72 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,10 +1,14 @@
-netcfg (1.90) UNRELEASED; urgency=low
+netcfg (1.91-SprezzOS1) unstable; urgency=low

+  [ Philipp Kern ]
   * netcfg.h: drop ip6-localnet (obsolete)
   * netcfg.h: drop ip6-mcastprefix (no need for it to be in /etc/hosts)
   * netcfg-common.c (is_layer3_qeth): fix off-by-one, introduce a constant

- -- Philipp Kern <pkern@debian.org>  Tue, 28 Aug 2012 23:57:14 +0200
+  [ Nick Black ]
+  * Rebase against new upstream version
+
+ -- Nick Black <nick.black@sprezzatech.com>  Sun, 02 Sep 2012 22:04:46 +0000
 
 netcfg (1.89) unstable; urgency=low
diff --git a/debian/netcfg-common.templates b/debian/netcfg-common.templates
index e964a5a..f1fa681 100644
--- a/debian/netcfg-common.templates
+++ b/debian/netcfg-common.templates
@@ -141,7 +141,7 @@ _Description: Failure of key exchange and association
 
 Template: netcfg/get_hostname
 Type: string
-Default: debian
+Default: sprezzabox
 # :sl1:
 _Description: Hostname:
  Please enter the hostname for this system.
diff --git a/netcfg-common.c b/netcfg-common.c
index 555d4d9..20aa3dc 100644
--- a/netcfg-common.c
+++ b/netcfg-common.c
@@ -995,7 +995,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char **hos
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             free(*hostname);
             *hostname = NULL;
         }
@@ -1023,7 +1023,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char **hos
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             free(*hostname);
             *hostname = NULL;
         } else {
