diff --git a/debian/changelog b/debian/changelog
index edfc2de..71b0c4e 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,8 +1,12 @@
-netcfg (1.96) UNRELEASED; urgency=low
+netcfg (1.97-SprezzOS1) unstable; urgency=low
 
+  [ Philipp Kern ]
   * Makefile: set DEB_HOST_ARCH{,_OS} conditionally
 
- -- Philipp Kern <pkern@debian.org>  Sat, 29 Sep 2012 17:03:57 +0200
+  [ Nick Black ]
+  * SprezzOS build
+
+ -- Nick Black <nick.black@sprezzatech.com>  Sun, 30 Sep 2012 11:23:09 +0000
 
 netcfg (1.95) unstable; urgency=low
 
diff --git a/debian/netcfg-common.templates b/debian/netcfg-common.templates
index c579a16..5e042db 100644
--- a/debian/netcfg-common.templates
+++ b/debian/netcfg-common.templates
@@ -142,7 +142,7 @@ _Description: Failure of key exchange and association
 
 Template: netcfg/get_hostname
 Type: string
-Default: debian
+Default: sprezzabox
 # :sl1:
 _Description: Hostname:
  Please enter the hostname for this system.
diff --git a/netcfg-common.c b/netcfg-common.c
index ac23be0..088c567 100644
--- a/netcfg-common.c
+++ b/netcfg-common.c
@@ -1038,7 +1038,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char *host
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             *hostname = '\0';
         }
 
@@ -1063,7 +1063,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char *host
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             *hostname = '\0';
         } else {
             break;
