diff --git a/debian/changelog b/debian/changelog
index 821f9ee..af6406b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+netcfg (1.89-SprezzOS1) unstable; urgency=low
+
+  * New upstream version 1.89
+
+ -- Nick Black <nick.black@sprezzatech.com>  Tue, 14 Aug 2012 20:45:38 -0400
+
 netcfg (1.89) UNRELEASED; urgency=low
 
   * Team upload
diff --git a/debian/netcfg-common.templates b/debian/netcfg-common.templates
index e964a5a..f1fa681 100644
--- a/debian/netcfg-common.templates
+++ b/debian/netcfg-common.templates
@@ -141,7 +141,7 @@ _Description: Failure of key exchange and association
 
 Template: netcfg/get_hostname
 Type: string
-Default: debian
+Default: sprezzabox
 # :sl1:
 _Description: Hostname:
  Please enter the hostname for this system.
diff --git a/netcfg-common.c b/netcfg-common.c
index 555d4d9..20aa3dc 100644
--- a/netcfg-common.c
+++ b/netcfg-common.c
@@ -995,7 +995,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char **hos
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             free(*hostname);
             *hostname = NULL;
         }
@@ -1023,7 +1023,7 @@ int netcfg_get_hostname(struct debconfclient *client, char *template, char **hos
                       "maxhostnamelen", buf);
             debconf_input(client, "high", "netcfg/invalid_hostname");
             debconf_go(client);
-            debconf_set(client, template, "debian");
+            debconf_set(client, template, "sprezzabox");
             free(*hostname);
             *hostname = NULL;
         } else {
