diff --git a/debian/changelog b/debian/changelog
index 0f7400e..45fa583 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,17 @@
+rootskel (1.100+SprezzOS12) unstable; urgency=high
+
+  * Get tangoize / VT interactions working via vicious hack
+  * Invoke "tangoize" from fbvfbterm; protect arguments
+  * Add "tangoize" script and install it to /usr/sbin
+  * Add -s and --font-height arguments to fbterm invocation
+  * Add fbvfbterm, sprezzos logo (sprezzos.png)
+  * Recursively build usr
+  * -n 'Terminus (TTF) -s 18 --font-height 18 solidifies borders
+  * Set TERM=fbterm. Use Tango-Dark palette on console.
+  * Open up tty7
+
+ -- Nick Black <nick.black@sprezzatech.com>  Mon, 20 Aug 2012 17:24:55 -0400
+
 rootskel (1.100) unstable; urgency=high
 
   [ Joey Hess ]
diff --git a/src/Makefile b/src/Makefile
index 10ef04f..a73d15c 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -6,6 +6,7 @@ subdirs = \
 	etc \
 	lib \
 	sbin \
+	usr \
 	var
 
 subdirs_empty = \
diff --git a/src/etc/inittab-linux b/src/etc/inittab-linux
index a7b8a23..c3e6d88 100644
--- a/src/etc/inittab-linux
+++ b/src/etc/inittab-linux
@@ -2,17 +2,20 @@
 # busybox init configuration for debian-installer
 
 # main rc script
-::sysinit:/sbin/reopen-console /sbin/debian-installer-startup
+::sysinit:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer-startup
 
 # main setup program
-::respawn:/sbin/reopen-console /sbin/debian-installer
+::respawn:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer
 
 # convenience shells
-tty2::askfirst:-/bin/sh
-tty3::askfirst:-/bin/sh
+tty2::askfirst:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
+tty3::askfirst:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
 
 # logging
-tty4::respawn:/usr/bin/tail -f /var/log/syslog
+tty4::respawn:/usr/bin/tail -f /var/log/syslog
+
+tty5::askfirst:-/bin/sh
+tty6::respawn:/usr/bin/tail -f /var/log/syslog
 
 # Stuff to do before rebooting
 ::ctrlaltdel:/sbin/shutdown > /dev/null 2>&1
diff --git a/src/sbin/Makefile b/src/sbin/Makefile
index dec554e..b96c231 100644
--- a/src/sbin/Makefile
+++ b/src/sbin/Makefile
@@ -9,7 +9,9 @@ files_exec = \
 	shutdown \
 	init:init-$(DEB_HOST_ARCH_OS) \
 	reopen-console:reopen-console-$(DEB_HOST_ARCH_OS) \
-	steal-ctty
+	steal-ctty \
+	fbvfbterm \
+	tangoize
 
 ifeq ($(DEB_HOST_ARCH_OS),linux)
   files_exec += \
