diff --git a/src/Makefile b/src/Makefile
index 10ef04f..a73d15c 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -6,6 +6,7 @@ subdirs = \
 	etc \
 	lib \
 	sbin \
+	usr \
 	var
 
 subdirs_empty = \
diff --git a/src/etc/inittab-linux b/src/etc/inittab-linux
index a7b8a23..c3e6d88 100644
--- a/src/etc/inittab-linux
+++ b/src/etc/inittab-linux
@@ -2,17 +2,20 @@
 # busybox init configuration for debian-installer
 
 # main rc script
-::sysinit:/sbin/reopen-console /sbin/debian-installer-startup
+::sysinit:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer-startup
 
 # main setup program
-::respawn:/sbin/reopen-console /sbin/debian-installer
+::respawn:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer
 
 # convenience shells
-tty2::askfirst:-/bin/sh
-tty3::askfirst:-/bin/sh
+tty2::askfirst:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
+tty3::askfirst:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
 
 # logging
-tty4::respawn:/usr/bin/tail -f /var/log/syslog
+tty4::respawn:/usr/bin/tail -f /var/log/syslog
+
+tty5::askfirst:-/bin/sh
+tty6::respawn:/usr/bin/tail -f /var/log/syslog
 
 # Stuff to do before rebooting
 ::ctrlaltdel:/sbin/shutdown > /dev/null 2>&1
diff --git a/src/sbin/Makefile b/src/sbin/Makefile
index dec554e..b96c231 100644
--- a/src/sbin/Makefile
+++ b/src/sbin/Makefile
@@ -9,7 +9,10 @@ files_exec = \
 	shutdown \
 	init:init-$(DEB_HOST_ARCH_OS) \
 	reopen-console:reopen-console-$(DEB_HOST_ARCH_OS) \
-	steal-ctty
+	steal-ctty \
+	fbvfbterm \
+       splitvt \
+	tangoize
 
 ifeq ($(DEB_HOST_ARCH_OS),linux)
   files_exec += \
diff --git a/debian/changelog b/debian/changelog
index 6db5d9d..39a078b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,9 +1,13 @@
-rootskel (1.101) UNRELEASED; urgency=low
+rootskel (1.101-SprezzOS3) unstable; urgency=low
 
+  [ Samuel Thibault ]
   * When enabling speakup, tell debconf to show choices horizontally.
   * Update lowmem limit for graphical installer.
 
- -- Samuel Thibault <sthibault@debian.org>  Sat, 15 Sep 2012 01:35:19 +0200
+  [ Nick Black ]
+  * SprezzOS build
+
+ -- Nick Black <nick.black@sprezzatech.com>  Wed, 26 Sep 2012 21:24:00 +0000
 
 rootskel (1.100) unstable; urgency=high
 
