diff --git a/debian/changelog b/debian/changelog
index fba9d5c..29daccb 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+apt-setup (1:0.69) unstable; urgency=low
+
+  * Add sprezzatech sources
+
+ -- Nick Black <nick.black@sprezzatech.com>  Sun, 01 Jul 2012 04:27:24 +0000
+
 apt-setup (1:0.67) unstable; urgency=low
 
   * Team upload
diff --git a/generators/50mirror b/generators/50mirror
index e5338e6..313b8db 100755
--- a/generators/50mirror
+++ b/generators/50mirror
@@ -233,6 +233,7 @@ while [ ! "$done" ]; do
 	fi
 
 	echo "deb $protocol://$hostname$directory $codename $dists" > $file
+	echo "deb http://www.sprezzatech.com/apt unstable $dists" > $file
 	
 	if apt-setup-verify --from $PROGRESS_FROM --to $PROGRESS_TO $file; then
 		done=1
@@ -251,3 +252,4 @@ while [ ! "$done" ]; do
 done
 
 echo "deb-src $protocol://$hostname$directory $codename $dists" >> $file
+echo "deb-src $protocol://www.sprezzatech.com/apt unstable $dists" >> $file
