diff --git a/debian/control b/debian/control
index 9d25334..ed8b6ce 100644
--- a/debian/control
+++ b/debian/control
@@ -11,6 +11,5 @@ Package: lilo-installer
 Architecture: i386 amd64
 Provides: bootable-system
 Depends: ${misc:Depends}, kernel-installer, di-utils (>= 1.15), di-utils-mapdevfs, fdisk-udeb, os-prober
-XB-Installer-Menu-Item: 7500
 Package-Type: udeb
 Description: Install LILO on a hard disk
