diff --git a/debian/control b/debian/control
index 728311f..50840c4 100644
--- a/debian/control
+++ b/debian/control
@@ -10,6 +10,6 @@ Vcs-Git: git://git.debian.org/d-i/partman-basicfilesystems.git
 Package: partman-basicfilesystems
 XC-Package-Type: udeb
 Architecture: all
-Depends: ${misc:Depends}, e2fsprogs-udeb, ext2-modules, fat-modules
+Depends: ${misc:Depends}, e2fsprogs-udeb, fat-modules
 Provides: partman-filesystem
 Description: Add to partman support for ext2, linux-swap, fat16 and fat32

