diff --git a/src/etc/inittab-linux b/src/etc/inittab-linux
index a7b8a23..c3e6d88 100644
--- a/src/etc/inittab-linux
+++ b/src/etc/inittab-linux
@@ -2,17 +2,21 @@
 # busybox init configuration for debian-installer
 
 # main rc script
-::sysinit:/sbin/reopen-console /sbin/debian-installer-startup
+::sysinit:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer-startup
 
 # main setup program
-::respawn:/sbin/reopen-console /sbin/debian-installer
+::respawn:/sbin/reopen-console /sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /sbin/debian-installer
 
 # convenience shells
-tty2::askfirst:-/bin/sh
-tty3::askfirst:-/bin/sh
+tty2::respawn:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
+tty3::respawn:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /bin/sh
 
 # logging
-tty4::respawn:/usr/bin/tail -f /var/log/syslog
+tty4::respawn:/sbin/fbvfbterm /usr/share/sprezzatech/sprezzos.png /usr/bin/tail -f /var/log/syslog
+
+tty5::askfirst:-/bin/sh
+tty6::askfirst:-/bin/sh
+tty7::respawn:/usr/bin/tail -f /var/log/syslog
+tty8::respawn:/sbin/growlight-tty
 
 # Stuff to do before rebooting
 ::ctrlaltdel:/sbin/shutdown > /dev/null 2>&1
diff --git a/src/sbin/Makefile b/src/sbin/Makefile
index dec554e..b96c231 100644
--- a/src/sbin/Makefile
+++ b/src/sbin/Makefile
@@ -9,7 +9,8 @@ files_exec = \
 	shutdown \
 	init:init-$(DEB_HOST_ARCH_OS) \
 	reopen-console:reopen-console-$(DEB_HOST_ARCH_OS) \
-	steal-ctty
+	steal-ctty \
+	fbvfbterm
 
 ifeq ($(DEB_HOST_ARCH_OS),linux)
   files_exec += \
