diff --git a/build/boot/x86/adgtk.cfg b/build/boot/x86/adgtk.cfg
index 26ad24b..e3b5714 100644
--- a/build/boot/x86/adgtk.cfg
+++ b/build/boot/x86/adgtk.cfg
@@ -1,12 +1,9 @@
-menu hshift 9
-menu width 58
-
 label expertgui
-	menu label Graphical expert install
+	menu label graphical expert install
 	kernel ${KERNEL}
-	append desktop=%desktop% priority=low ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- ${CONSOLE}
+	append desktop=%desktop% vga=788 priority=low ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- ${CONSOLE}
 include ${SYSDIR}rqgtk.cfg
 label autogui
-	menu label Graphical automated install
+	menu label graphical automated install
 	kernel ${KERNEL}
-	append desktop=%desktop% auto=true priority=critical ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
+	append desktop=%desktop% vga=788 auto=true priority=critical ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/adtxt.cfg b/build/boot/x86/adtxt.cfg
index 268d456..a8da86a 100644
--- a/build/boot/x86/adtxt.cfg
+++ b/build/boot/x86/adtxt.cfg
@@ -1,9 +1,9 @@
 label expert
-	menu label ^Expert install
+	menu label ^expert install
 	kernel ${KERNEL}
 	append desktop=%desktop% priority=low ${VIDEO_MODE} initrd=${INITRD} -- ${CONSOLE}
 include ${SYSDIR}rqtxt.cfg
 label auto
-	menu label ^Automated install
+	menu label ^automated install
 	kernel ${KERNEL}
 	append desktop=%desktop% auto=true priority=critical ${VIDEO_MODE} initrd=${INITRD} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/amdadtxt.cfg b/build/boot/x86/amdadtxt.cfg
index f3863a6..0a6b243 100644
--- a/build/boot/x86/amdadtxt.cfg
+++ b/build/boot/x86/amdadtxt.cfg
@@ -1,6 +1,3 @@
-menu hshift 9
-menu width 58
-
 label amd64-expert
 	menu label 64 bit expert install
 	kernel ${AMD_KERNEL}
diff --git a/build/boot/x86/desktop/adgtkdt.cfg b/build/boot/x86/desktop/adgtkdt.cfg
index 41e4ad8..aa3c90e 100644
--- a/build/boot/x86/desktop/adgtkdt.cfg
+++ b/build/boot/x86/desktop/adgtkdt.cfg
@@ -2,10 +2,10 @@ menu hshift 9
 menu width 58
 
 label expertgui-%desktop%
-	menu label Graphical expert install
+	menu label graphical expert install
 	kernel ${KERNEL}
 	append desktop=%desktop% priority=low ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- ${CONSOLE}
 label autogui-%desktop%
-	menu label Graphical automated install
+	menu label graphical automated install
 	kernel ${KERNEL}
 	append desktop=%desktop% auto=true priority=critical ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/desktop/gtkdt.cfg b/build/boot/x86/desktop/gtkdt.cfg
index 282ba03..4d5a6df 100644
--- a/build/boot/x86/desktop/gtkdt.cfg
+++ b/build/boot/x86/desktop/gtkdt.cfg
@@ -1,4 +1,4 @@
 label installgui-%desktop%
-	menu label ^Graphical install
+	menu label ^graphical install
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/desktop/menu.cfg b/build/boot/x86/desktop/menu.cfg
index 7752600..5b15843 100644
--- a/build/boot/x86/desktop/menu.cfg
+++ b/build/boot/x86/desktop/menu.cfg
@@ -3,11 +3,11 @@ include ${SYSDIR}%desktop%/amdtxtdt.cfg
 include ${SYSDIR}%desktop%/gtkdt.cfg
 include ${SYSDIR}%desktop%/amdgtkdt.cfg
 menu begin advanced-%desktop%
-    menu label Advanced options
+    menu label advanced options
 	menu title %dt-name% advanced options
 	include ${SYSDIR}stdmenu.cfg
 	label dtmenu-%desktop%
-		menu label ^Back..
+		menu label ^back..
 		menu exit
 	include ${SYSDIR}%desktop%/adtxtdt.cfg
 	include ${SYSDIR}%desktop%/amdatdt.cfg
@@ -15,7 +15,7 @@ menu begin advanced-%desktop%
 	include ${SYSDIR}%desktop%/amdagdt.cfg
 menu end
 label help-%desktop%
-	menu label ^Help
+	menu label ^help
 	text help
    Display help screens; type 'menu' at boot prompt to return to this menu
 	endtext
diff --git a/build/boot/x86/desktop/txtdt.cfg b/build/boot/x86/desktop/txtdt.cfg
index 7ef5b82..074f588 100644
--- a/build/boot/x86/desktop/txtdt.cfg
+++ b/build/boot/x86/desktop/txtdt.cfg
@@ -1,6 +1,6 @@
 #default install
 label install-%desktop%
-	menu label ^Install
+	menu label ^install
 	menu default
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE} initrd=${INITRD} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/f1.txt b/build/boot/x86/f1.txt
index 53aa5a2..2a9a85d 100644
--- a/build/boot/x86/f1.txt
+++ b/build/boot/x86/f1.txt
@@ -1,6 +1,6 @@
-                  0fWelcome to Debian GNU/Linux!07                                09F107
+                  0fwelcome to sprezzos07                                09F107
 
-This is a Debian ${DEBIAN_VERSION} installation ${MEDIA_TYPE}.
+This is a SprezzOS ${DEBIAN_VERSION} installation ${MEDIA_TYPE}.
 It was built on ${BUILD_DATE}.
 
 0fHELP INDEX07
@@ -8,7 +8,7 @@ It was built on ${BUILD_DATE}.
 0fKEY    TOPIC07
 
 <09F107>   This page, the help index.
-<09F207>   Prerequisites for installing Debian.
+<09F207>   Prerequisites for installing SprezzOS.
 <09F307>   Boot methods for special ways of using this ${MEDIA_TYPE}
 <09F407>   Additional boot methods; rescue mode.
 <09F507>   Special boot parameters, overview.
diff --git a/build/boot/x86/f10.txt b/build/boot/x86/f10.txt
index 2166ea7..d72e15d 100644
--- a/build/boot/x86/f10.txt
+++ b/build/boot/x86/f10.txt
@@ -1,13 +1,13 @@
 0fCOPYRIGHTS AND WARRANTIES07                                                    09F1007
 
-Debian GNU/Linux is Copyright (C) 1993-2009 Software in the Public Interest,
+SprezzOS is Copyright (C) 2012 Sprezzatura Computing Corporation, Inc.,
 and others.
 
-The Debian GNU/Linux system is freely redistributable. After installation,
-the exact distribution terms for each package are described in the
-corresponding file /usr/share/doc/0bpackagename07/copyright.
+SprezzOS is freely redistributable. After installation, the exact distribution
+terms for each package are described in the corresponding file
+/usr/share/doc/0bpackagename07/copyright.
 
-Debian GNU/Linux comes with 0fABSOLUTELY NO WARRANTY07, to the extent
+SprezzOS comes with 0fABSOLUTELY NO WARRANTY07, to the extent
 permitted by applicable law.
 
 
diff --git a/build/boot/x86/f2.txt b/build/boot/x86/f2.txt
index 14cc7a2..e4d06d2 100644
--- a/build/boot/x86/f2.txt
+++ b/build/boot/x86/f2.txt
@@ -1,16 +1,16 @@
-0fPREREQUISITES FOR INSTALLING DEBIAN07                                           09F207
+0fPREREQUISITES FOR INSTALLING SPREZZOS07                                           09F207
 
-You must have at least 44 megabytes of RAM to use this Debian installer.
+You must have at least 44 megabytes of RAM to use this SprezzOS installer.
 
 You should have space on your hard disk to create a new disk partition
 of at least 500 megabytes to install the base system. You'll need more
 disk space to install additional packages, depending on what you wish
-to do with your new Debian system.
+to do with your new SprezzOS system.
 
 See the Installation Guide or the FAQ for more information; both
-documents are available at the Debian web site, 0fhttp://www.debian.org/07
+documents are available at the SprezzOS web site, 0fhttp://www.sprezzos.com/07
 
-Thank you for choosing Debian!
+Thank you for choosing SprezzOS!
 
 
 
diff --git a/build/boot/x86/f8.txt b/build/boot/x86/f8.txt
index bee2894..a15f415 100644
--- a/build/boot/x86/f8.txt
+++ b/build/boot/x86/f8.txt
@@ -12,7 +12,7 @@ Set keyboard map                        0fbootkbd=es07
 Select the kde or xfce desktops         0fdesktop=kde07
 
 Accessibility options (last 2 options not available for all images):
-  Use high contrast theme               0ftheme=dark07
+  Use low contrast theme                0ftheme=light07
   Use Braille tty                       0fbrltty=driver,device,texttable07
   Use Speakup                           0fspeakup.synth=driver07
 
diff --git a/build/boot/x86/f9.txt b/build/boot/x86/f9.txt
index ef743ef..189af7c 100644
--- a/build/boot/x86/f9.txt
+++ b/build/boot/x86/f9.txt
@@ -1,6 +1,6 @@
 0fGETTING HELP07                                                                  09F907
 
-If you can't install Debian, don't despair! The Debian team is ready to
+If you can't install SprezzOS, don't despair! The SprezzOS team is ready to
 help you! We are especially interested in hearing about installation
 problems, because in general they don't happen to only 0fone07 person.
 We've either already heard about your particular problem and can dispense a
@@ -9,7 +9,7 @@ and the next user who comes up with the same problem will profit from your
 experience!
 
 See the Installation Guide or the FAQ for more information; both
-documents are available at the Debian web site, 0fhttp://www.debian.org/07
+documents are available at the SprezzOS web site, 0fhttp://www.sprezzos.com/07
 
 
 
diff --git a/build/boot/x86/gtk.cfg b/build/boot/x86/gtk.cfg
index 3cfb88f..77a64f8 100644
--- a/build/boot/x86/gtk.cfg
+++ b/build/boot/x86/gtk.cfg
@@ -1,4 +1,4 @@
 label installgui
-	menu label ^Graphical install
+	menu label ^graphical install
 	kernel ${KERNEL}
-	append desktop=%desktop% ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
+	append desktop=%desktop% vga=788 ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} -- quiet ${CONSOLE}
diff --git a/build/boot/x86/menu.cfg b/build/boot/x86/menu.cfg
index 721f74f..7fac20b 100644
--- a/build/boot/x86/menu.cfg
+++ b/build/boot/x86/menu.cfg
@@ -1,31 +1,31 @@
-menu hshift 13
-menu width 60
+menu hshift 45
+menu width 35
+menu margin 1
 
-menu title Debian GNU/Linux installer boot menu${BEEP}
+menu title      legacy bios-mode installer ${BEEP}
 include ${SYSDIR}stdmenu.cfg
 include ${SYSDIR}txt.cfg
 include ${SYSDIR}amdtxt.cfg
 include ${SYSDIR}gtk.cfg
 include ${SYSDIR}amdgtk.cfg
-menu begin advanced
-	menu title Advanced options
-	include ${SYSDIR}stdmenu.cfg
-	label mainmenu
-		menu label ^Back..
-		menu exit
-	include ${SYSDIR}adtxt.cfg
-	include ${SYSDIR}amdadtxt.cfg
-	include ${SYSDIR}adgtk.cfg
-	include ${SYSDIR}amdadgtk.cfg
-	include ${SYSDIR}dtmenu.cfg
-menu end
-label help
-	menu label ^Help
-	text help
-   Display help screens; type 'menu' at boot prompt to return to this menu
-	endtext
-	config ${SYSDIR}prompt.cfg
+include ${SYSDIR}adtxt.cfg
+include ${SYSDIR}amdadtxt.cfg
+include ${SYSDIR}adgtk.cfg
+include ${SYSDIR}amdadgtk.cfg
+include ${SYSDIR}dtmenu.cfg
 include ${SYSDIR}spk.cfg
 include ${SYSDIR}amdspk.cfg
 include ${SYSDIR}spkgtk.cfg
 include ${SYSDIR}amdspgtk.cfg
+label help
+	menu label ^help
+	text help
+   display help screens. type 'menu' at boot prompt to return here.
+	endtext
+	config ${SYSDIR}prompt.cfg
+label memtest86
+	menu label ^memtest86
+	kernel /memtest86+.bin
+	text help
+   run memory tester. you will need to reboot to exit.
+	endtext
diff --git a/build/boot/x86/rqgtk.cfg b/build/boot/x86/rqgtk.cfg
index 8a88e32..dfa78ed 100644
--- a/build/boot/x86/rqgtk.cfg
+++ b/build/boot/x86/rqgtk.cfg
@@ -1,4 +1,4 @@
 label rescuegui
-	menu label Graphical rescue mode
+	menu label graphical rescue mode
 	kernel ${KERNEL}
 	append ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} rescue/enable=true -- quiet  ${CONSOLE}
diff --git a/build/boot/x86/rqtxt.cfg b/build/boot/x86/rqtxt.cfg
index a511c35..e537b83 100644
--- a/build/boot/x86/rqtxt.cfg
+++ b/build/boot/x86/rqtxt.cfg
@@ -1,4 +1,4 @@
 label rescue
-	menu label ^Rescue mode
+	menu label ^rescue mode
 	kernel ${KERNEL}
 	append ${VIDEO_MODE} initrd=${INITRD} rescue/enable=true -- quiet ${CONSOLE}
diff --git a/build/boot/x86/spk.cfg b/build/boot/x86/spk.cfg
index 270a415..da9db6e 100644
--- a/build/boot/x86/spk.cfg
+++ b/build/boot/x86/spk.cfg
@@ -1,4 +1,4 @@
 label installspk
-	menu label Install with ^speech synthesis
+	menu label install with ^speech synthesis
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE} initrd=${INITRD} speakup.synth=soft -- quiet ${CONSOLE}
diff --git a/build/boot/x86/spkgtk.cfg b/build/boot/x86/spkgtk.cfg
index 8212aec..a36fc5b 100644
--- a/build/boot/x86/spkgtk.cfg
+++ b/build/boot/x86/spkgtk.cfg
@@ -1,4 +1,4 @@
 label installspk
-	menu label Install with ^speech synthesis
+	menu label install with ^speech synthesis
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE} initrd=${INITRD_GTK} speakup.synth=soft -- quiet ${CONSOLE}
diff --git a/build/boot/x86/stdmenu.cfg b/build/boot/x86/stdmenu.cfg
index 4a212b6..f941778 100644
--- a/build/boot/x86/stdmenu.cfg
+++ b/build/boot/x86/stdmenu.cfg
@@ -4,15 +4,15 @@ menu color border	* #00000000 #00000000 none
 menu color sel		* #ffffffff #76a1d0ff *
 menu color hotsel	1;7;37;40 #ffffffff #76a1d0ff *
 menu color tabmsg	* #ffffffff #00000000 *
-menu color help		37;40 #ffdddd00 #00000000 none
+menu color help		* #88888888 #00000000 *
 # XXX When adjusting vshift, take care that rows is set to a small
 # enough value so any possible menu will fit on the screen,
 # rather than falling off the bottom.
-menu vshift 12
-menu rows 10
+menu vshift 4
+menu rows 17
 menu helpmsgrow 15
 # The command line must be at least one line from the bottom.
-menu cmdlinerow 16
-menu timeoutrow 16
-menu tabmsgrow 18
-menu tabmsg Press ENTER to boot or TAB to edit a menu entry
+menu cmdlinerow 20
+menu timeoutrow 20
+menu tabmsgrow 20
+menu tabmsg enter selects. tab configures.
diff --git a/build/boot/x86/txt.cfg b/build/boot/x86/txt.cfg
index d15af36..4a4a728 100644
--- a/build/boot/x86/txt.cfg
+++ b/build/boot/x86/txt.cfg
@@ -1,6 +1,6 @@
 default install
 label install
-	menu label ^Install
+	menu label ^install
 	menu default
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE} initrd=${INITRD} -- quiet ${CONSOLE}
diff --git a/build/config/amd64.cfg b/build/config/amd64.cfg
index 343d947..0bb5976 100644
--- a/build/config/amd64.cfg
+++ b/build/config/amd64.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-KERNELVERSION = 3.2.0-2-amd64
+KERNELVERSION = 3.4-1-amd64
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
diff --git a/build/config/common b/build/config/common
index 6ebfc9e..27fb3ff 100644
--- a/build/config/common
+++ b/build/config/common
@@ -20,11 +20,11 @@ EXTRAS ?=
 
 # This variable can be used to add a preseed file to the initrd itself.
 # It will be loaded by the initrd-preseed udeb.
-PRESEED = 
+PRESEED = SprezzOS.preseed
 
 # The library reducer to use. Set to mklibs (to build with library reduction)
 # or mklibs-copy (to build without library reduction).
-MKLIBS = mklibs
+MKLIBS = mklibs-copy
 
 # Normally the sources.list for building d-i will be derived from your
 # "normal" sources.list. However, you can specify a mirror here to override
@@ -37,13 +37,13 @@ MKLIBS = mklibs
 DPKG_UNPACK_OPTIONS = --force-overwrite
 
 # The codename of the Debian release that should be installed by default.
-DEBIAN_RELEASE = wheezy
+DEBIAN_RELEASE = unstable
 
 # The version (number and codename) of the Debian release targeted by the
 # installation images. (Only the codename is used while the version number
 # has not yet been determined.)
-DEBIAN_VERSION = 7.0 (wheezy)
-DEBIAN_VERSION ?= wheezy
+DEBIAN_VERSION = 1.0 (cray)
+DEBIAN_VERSION ?= cray
 
 # Define here from what release (codename!) components (udebs) should be taken
 # to build debian-installer. By default this is set to 'unstable' for building
@@ -57,12 +57,12 @@ UDEB_COMPONENTS ?= main/debian-installer
 BUILD_DATE ?= $(shell date -u '+%Y%m%d-%H:%M')
 
 # Produce a beep at boot menu
-BOOTMENU_BEEP ?= n
+BOOTMENU_BEEP ?= y
 
 # x86: configure syslinux to have tasksel install a different desktop
 # environment than the default (gnome).
 # Possible values: kde, xfce, lxde
-DESKTOP = 
+DESKTOP = xfce
 
 # Filesystem type for the initrd, currently valid values are ext2, cramfs,
 # and initramfs.
@@ -73,7 +73,7 @@ INITRD_FS = initramfs
 FLOPPYDEV = /dev/fd0
 
 # What qemu to use, with any options.
-QEMU = qemu
+QEMU = kvm
 
 # media where images may be written to (via dd).
 WRITE_MEDIA = floppy
@@ -84,7 +84,7 @@ WRITE_MEDIA = floppy
 TYPE = $(MEDIUM)
 
 # Used to write a /etc/lsb-release file in the installer initrd.
-LSB_DISTRIB_ID=Debian
+LSB_DISTRIB_ID=SprezzOS
 LSB_DISTRIB_DESCRIPTION="Debian $(shell uname -o) installer"
 # Inlude the build date of the installer in here.
 LSB_DISTRIB_RELEASE="$(DEBIAN_VERSION) - installer build $(BUILD_DATE)"
diff --git a/build/config/i386.cfg b/build/config/i386.cfg
index ba92623..0c17ab6 100644
--- a/build/config/i386.cfg
+++ b/build/config/i386.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-BASEVERSION = 3.2.0-2
+BASEVERSION = 3.3.0-2
 KERNELVERSION = $(BASEVERSION)-486
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
diff --git a/build/config/ia64.cfg b/build/config/ia64.cfg
index 765be19..657e4d4 100644
--- a/build/config/ia64.cfg
+++ b/build/config/ia64.cfg
@@ -5,7 +5,7 @@ MEDIUM_SUPPORTED = cdrom netboot
 
 # The version of the kernel to use.
 KERNELMAJOR = 2.6
-KERNELVERSION = 3.2.0-2-itanium
+KERNELVERSION = 3.4-1-itanium
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
 KERNELIMAGEVERSION = $(KERNELVERSION)
@@ -17,7 +17,7 @@ INITRD_FS = initramfs
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 
 arch_tree:
diff --git a/build/config/x86.cfg b/build/config/x86.cfg
index a0f9976..bb1cb92 100644
--- a/build/config/x86.cfg
+++ b/build/config/x86.cfg
@@ -8,7 +8,7 @@
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 # The image to use for a syslinux splash screen.
 SPLASH_PNG=boot/x86/pics/klowner.png
diff --git a/build/pkg-lists/hd-media/amd64.cfg b/build/pkg-lists/hd-media/amd64.cfg
index 415d281..5b59baa 100644
--- a/build/pkg-lists/hd-media/amd64.cfg
+++ b/build/pkg-lists/hd-media/amd64.cfg
@@ -24,10 +24,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/i386.cfg b/build/pkg-lists/hd-media/i386.cfg
index 415d281..5b59baa 100644
--- a/build/pkg-lists/hd-media/i386.cfg
+++ b/build/pkg-lists/hd-media/i386.cfg
@@ -24,10 +24,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/powerpc.cfg b/build/pkg-lists/hd-media/powerpc.cfg
index 97faa81..0355258 100644
--- a/build/pkg-lists/hd-media/powerpc.cfg
+++ b/build/pkg-lists/hd-media/powerpc.cfg
@@ -19,10 +19,6 @@ sata-modules-${kernel:Version}
 # all the filesystems (feel free to add more!)
 hfs-modules-${kernel:Version}
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
