diff --git a/build/config/amd64.cfg b/build/config/amd64.cfg
index 343d947..0bb5976 100644
--- a/build/config/amd64.cfg
+++ b/build/config/amd64.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-KERNELVERSION = 3.2.0-2-amd64
+KERNELVERSION = 3.4.3-1-amd64
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
diff --git a/build/config/common b/build/config/common
index 6ebfc9e..27fb3ff 100644
--- a/build/config/common
+++ b/build/config/common
@@ -20,11 +20,11 @@ EXTRAS ?=
 
 # This variable can be used to add a preseed file to the initrd itself.
 # It will be loaded by the initrd-preseed udeb.
-PRESEED = 
+PRESEED = SprezzOS.preseed
 
 # The library reducer to use. Set to mklibs (to build with library reduction)
 # or mklibs-copy (to build without library reduction).
-MKLIBS = mklibs
+MKLIBS = mklibs-copy
 
 # Normally the sources.list for building d-i will be derived from your
 # "normal" sources.list. However, you can specify a mirror here to override
@@ -37,13 +37,13 @@ MKLIBS = mklibs
 DPKG_UNPACK_OPTIONS = --force-overwrite
 
 # The codename of the Debian release that should be installed by default.
-DEBIAN_RELEASE = wheezy
+DEBIAN_RELEASE = unstable
 
 # The version (number and codename) of the Debian release targeted by the
 # installation images. (Only the codename is used while the version number
 # has not yet been determined.)
-DEBIAN_VERSION = 7.0 (wheezy)
-DEBIAN_VERSION ?= wheezy
+DEBIAN_VERSION = 1.0 (cray)
+DEBIAN_VERSION ?= cray
 
 # Define here from what release (codename!) components (udebs) should be taken
 # to build debian-installer. By default this is set to 'unstable' for building
@@ -57,12 +57,12 @@ UDEB_COMPONENTS ?= main/debian-installer
 BUILD_DATE ?= $(shell date -u '+%Y%m%d-%H:%M')
 
 # Produce a beep at boot menu
-BOOTMENU_BEEP ?= n
+BOOTMENU_BEEP ?= y
 
 # x86: configure syslinux to have tasksel install a different desktop
 # environment than the default (gnome).
 # Possible values: kde, xfce, lxde
-DESKTOP = 
+DESKTOP = xfce
 
 # Filesystem type for the initrd, currently valid values are ext2, cramfs,
 # and initramfs.
@@ -73,7 +73,7 @@ INITRD_FS = initramfs
 FLOPPYDEV = /dev/fd0
 
 # What qemu to use, with any options.
-QEMU = qemu
+QEMU = kvm
 
 # media where images may be written to (via dd).
 WRITE_MEDIA = floppy
@@ -84,7 +84,7 @@ WRITE_MEDIA = floppy
 TYPE = $(MEDIUM)
 
 # Used to write a /etc/lsb-release file in the installer initrd.
-LSB_DISTRIB_ID=Debian
+LSB_DISTRIB_ID=SprezzOS
 LSB_DISTRIB_DESCRIPTION="Debian $(shell uname -o) installer"
 # Inlude the build date of the installer in here.
 LSB_DISTRIB_RELEASE="$(DEBIAN_VERSION) - installer build $(BUILD_DATE)"
diff --git a/build/config/i386.cfg b/build/config/i386.cfg
index ba92623..0c17ab6 100644
--- a/build/config/i386.cfg
+++ b/build/config/i386.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-BASEVERSION = 3.2.0-2
+BASEVERSION = 3.4.3-1
 KERNELVERSION = $(BASEVERSION)-486
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
diff --git a/build/config/ia64.cfg b/build/config/ia64.cfg
index 765be19..657e4d4 100644
--- a/build/config/ia64.cfg
+++ b/build/config/ia64.cfg
@@ -5,7 +5,7 @@ MEDIUM_SUPPORTED = cdrom netboot
 
 # The version of the kernel to use.
 KERNELMAJOR = 2.6
-KERNELVERSION = 3.2.0-2-itanium
+KERNELVERSION = 3.4.3-1-itanium
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
 KERNELIMAGEVERSION = $(KERNELVERSION)
@@ -17,7 +17,7 @@ INITRD_FS = initramfs
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 
 arch_tree:
diff --git a/build/config/x86.cfg b/build/config/x86.cfg
index a0f9976..bb1cb92 100644
--- a/build/config/x86.cfg
+++ b/build/config/x86.cfg
@@ -8,7 +8,7 @@
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 # The image to use for a syslinux splash screen.
 SPLASH_PNG=boot/x86/pics/klowner.png
diff --git a/build/pkg-lists/hd-media/amd64.cfg b/build/pkg-lists/hd-media/amd64.cfg
index 415d281..5b59baa 100644
--- a/build/pkg-lists/hd-media/amd64.cfg
+++ b/build/pkg-lists/hd-media/amd64.cfg
@@ -24,10 +24,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/i386.cfg b/build/pkg-lists/hd-media/i386.cfg
index 415d281..5b59baa 100644
--- a/build/pkg-lists/hd-media/i386.cfg
+++ b/build/pkg-lists/hd-media/i386.cfg
@@ -24,10 +24,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/powerpc.cfg b/build/pkg-lists/hd-media/powerpc.cfg
index 97faa81..0355258 100644
--- a/build/pkg-lists/hd-media/powerpc.cfg
+++ b/build/pkg-lists/hd-media/powerpc.cfg
@@ -19,10 +19,6 @@ sata-modules-${kernel:Version}
 # all the filesystems (feel free to add more!)
 hfs-modules-${kernel:Version}
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
