diff --git a/build/config/amd64/cdrom.cfg b/build/config/amd64/cdrom.cfg
index 5678ba5..ec36871 100644
--- a/build/config/amd64/cdrom.cfg
+++ b/build/config/amd64/cdrom.cfg
@@ -1,5 +1,5 @@
 # el-torito is too large at the moment, so is disabled.
-FLAVOUR_SUPPORTED = isolinux gtk #el-torito
+FLAVOUR_SUPPORTED = isolinux #gtk el-torito
 
 MEDIA_TYPE = CD-ROM
 
diff --git a/build/config/amd64/cdrom/isolinux.cfg b/build/config/amd64/cdrom/isolinux.cfg
index 3d06000..a749f4e 100644
--- a/build/config/amd64/cdrom/isolinux.cfg
+++ b/build/config/amd64/cdrom/isolinux.cfg
@@ -9,6 +9,6 @@ MANIFEST-DEBIAN_CD_INFO = "isolinux config files for CD"
 TYPE = cdrom/isolinux
 
 # Add the gtk images on to the isolinux info.
-INITRD_GTK = dest/cdrom/gtk/initrd.gz
+#INITRD_GTK = dest/cdrom/gtk/initrd.gz
 
-EXTRATARGETS = build_cdrom_gtk
+#EXTRATARGETS = build_cdrom_gtk
diff --git a/build/config/amd64/hd-media.cfg b/build/config/amd64/hd-media.cfg
index 4aa3977..38eee9d 100644
--- a/build/config/amd64/hd-media.cfg
+++ b/build/config/amd64/hd-media.cfg
@@ -1,4 +1,4 @@
-FLAVOUR_SUPPORTED = "" gtk
+FLAVOUR_SUPPORTED = ""
 
 # Not really a floppy, this is a 956 MiB image, large enough to put a
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
@@ -13,8 +13,8 @@ GZIPPED = .gz
 EXTRANAME = hd-media/
 
 # Add the gtk images on to the bootable image.
-EXTRATARGETS = build_hd-media_gtk
-INITRD_GTK = dest/hd-media/gtk/initrd.gz
+#EXTRATARGETS = build_hd-media_gtk
+#INITRD_GTK = dest/hd-media/gtk/initrd.gz
 TARGET = $(KERNEL) $(INITRD) $(BOOT)
 
 MANIFEST-BOOT = "1 gb image (compressed) for USB memory stick"
diff --git a/build/config/i386/cdrom.cfg b/build/config/i386/cdrom.cfg
index 5678ba5..ec36871 100644
--- a/build/config/i386/cdrom.cfg
+++ b/build/config/i386/cdrom.cfg
@@ -1,5 +1,5 @@
 # el-torito is too large at the moment, so is disabled.
-FLAVOUR_SUPPORTED = isolinux gtk #el-torito
+FLAVOUR_SUPPORTED = isolinux #gtk el-torito
 
 MEDIA_TYPE = CD-ROM
 
diff --git a/build/config/i386/hd-media.cfg b/build/config/i386/hd-media.cfg
index 4aa3977..38eee9d 100644
--- a/build/config/i386/hd-media.cfg
+++ b/build/config/i386/hd-media.cfg
@@ -1,4 +1,4 @@
-FLAVOUR_SUPPORTED = "" gtk
+FLAVOUR_SUPPORTED = ""
 
 # Not really a floppy, this is a 956 MiB image, large enough to put a
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
@@ -13,8 +13,8 @@ GZIPPED = .gz
 EXTRANAME = hd-media/
 
 # Add the gtk images on to the bootable image.
-EXTRATARGETS = build_hd-media_gtk
-INITRD_GTK = dest/hd-media/gtk/initrd.gz
+#EXTRATARGETS = build_hd-media_gtk
+#INITRD_GTK = dest/hd-media/gtk/initrd.gz
 TARGET = $(KERNEL) $(INITRD) $(BOOT)
 
 MANIFEST-BOOT = "1 gb image (compressed) for USB memory stick"
diff --git a/build/pkg-lists/exclude b/build/pkg-lists/exclude
index 69fdc43..f190d14 100644
--- a/build/pkg-lists/exclude
+++ b/build/pkg-lists/exclude
@@ -18,3 +18,6 @@ libnewt0.52 -
 # some arches and versions of glibc link against libgcc1,
 # it will be pulled in via library reduction
 libgcc1 -
+
+lilo-installer -
+grub-installer -
