diff --git a/build/config/amd64.cfg b/build/config/amd64.cfg
index 397db3f..2ab43a6 100644
--- a/build/config/amd64.cfg
+++ b/build/config/amd64.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-KERNELVERSION = 3.2.0-3-amd64
+KERNELVERSION = 3.5.1-1-amd64
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
diff --git a/build/config/amd64/cdrom.cfg b/build/config/amd64/cdrom.cfg
index 5678ba5..ec36871 100644
--- a/build/config/amd64/cdrom.cfg
+++ b/build/config/amd64/cdrom.cfg
@@ -1,5 +1,5 @@
 # el-torito is too large at the moment, so is disabled.
-FLAVOUR_SUPPORTED = isolinux gtk #el-torito
+FLAVOUR_SUPPORTED = isolinux #gtk el-torito
 
 MEDIA_TYPE = CD-ROM
 
diff --git a/build/config/amd64/cdrom/isolinux.cfg b/build/config/amd64/cdrom/isolinux.cfg
index 3d06000..a749f4e 100644
--- a/build/config/amd64/cdrom/isolinux.cfg
+++ b/build/config/amd64/cdrom/isolinux.cfg
@@ -9,6 +9,6 @@ MANIFEST-DEBIAN_CD_INFO = "isolinux config files for CD"
 TYPE = cdrom/isolinux
 
 # Add the gtk images on to the isolinux info.
-INITRD_GTK = dest/cdrom/gtk/initrd.gz
+#INITRD_GTK = dest/cdrom/gtk/initrd.gz
 
-EXTRATARGETS = build_cdrom_gtk
+#EXTRATARGETS = build_cdrom_gtk
diff --git a/build/config/amd64/hd-media.cfg b/build/config/amd64/hd-media.cfg
index 4aa3977..38eee9d 100644
--- a/build/config/amd64/hd-media.cfg
+++ b/build/config/amd64/hd-media.cfg
@@ -1,4 +1,4 @@
-FLAVOUR_SUPPORTED = "" gtk
+FLAVOUR_SUPPORTED = ""
 
 # Not really a floppy, this is a 956 MiB image, large enough to put a
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
@@ -13,8 +13,8 @@ GZIPPED = .gz
 EXTRANAME = hd-media/
 
 # Add the gtk images on to the bootable image.
-EXTRATARGETS = build_hd-media_gtk
-INITRD_GTK = dest/hd-media/gtk/initrd.gz
+#EXTRATARGETS = build_hd-media_gtk
+#INITRD_GTK = dest/hd-media/gtk/initrd.gz
 TARGET = $(KERNEL) $(INITRD) $(BOOT)
 
 MANIFEST-BOOT = "1 gb image (compressed) for USB memory stick"
diff --git a/build/config/common b/build/config/common
index 6ebfc9e..b03d99b 100644
--- a/build/config/common
+++ b/build/config/common
@@ -20,11 +20,11 @@ EXTRAS ?=
 
 # This variable can be used to add a preseed file to the initrd itself.
 # It will be loaded by the initrd-preseed udeb.
-PRESEED = 
+PRESEED = SprezzOS.preseed
 
 # The library reducer to use. Set to mklibs (to build with library reduction)
 # or mklibs-copy (to build without library reduction).
-MKLIBS = mklibs
+MKLIBS = mklibs-copy
 
 # Normally the sources.list for building d-i will be derived from your
 # "normal" sources.list. However, you can specify a mirror here to override
@@ -37,13 +37,13 @@ MKLIBS = mklibs
 DPKG_UNPACK_OPTIONS = --force-overwrite
 
 # The codename of the Debian release that should be installed by default.
-DEBIAN_RELEASE = wheezy
+DEBIAN_RELEASE = unstable
 
 # The version (number and codename) of the Debian release targeted by the
 # installation images. (Only the codename is used while the version number
 # has not yet been determined.)
-DEBIAN_VERSION = 7.0 (wheezy)
-DEBIAN_VERSION ?= wheezy
+DEBIAN_VERSION = 0.0 (turing)
+DEBIAN_VERSION ?= turing
 
 # Define here from what release (codename!) components (udebs) should be taken
 # to build debian-installer. By default this is set to 'unstable' for building
@@ -57,12 +57,12 @@ UDEB_COMPONENTS ?= main/debian-installer
 BUILD_DATE ?= $(shell date -u '+%Y%m%d-%H:%M')
 
 # Produce a beep at boot menu
-BOOTMENU_BEEP ?= n
+BOOTMENU_BEEP ?= y
 
 # x86: configure syslinux to have tasksel install a different desktop
 # environment than the default (gnome).
 # Possible values: kde, xfce, lxde
-DESKTOP = 
+DESKTOP = xfce
 
 # Filesystem type for the initrd, currently valid values are ext2, cramfs,
 # and initramfs.
@@ -70,13 +70,13 @@ DESKTOP =
 INITRD_FS = initramfs
 
 # What device to write floppies on
-FLOPPYDEV = /dev/fd0
+#FLOPPYDEV = /dev/fd0
 
 # What qemu to use, with any options.
-QEMU = qemu
+QEMU = kvm
 
 # media where images may be written to (via dd).
-WRITE_MEDIA = floppy
+#WRITE_MEDIA = floppy
 
 # HACK Alert:
 # pkg-lists is still sorted by type, so we have to provide
@@ -84,7 +84,7 @@ WRITE_MEDIA = floppy
 TYPE = $(MEDIUM)
 
 # Used to write a /etc/lsb-release file in the installer initrd.
-LSB_DISTRIB_ID=Debian
+LSB_DISTRIB_ID=SprezzOS
 LSB_DISTRIB_DESCRIPTION="Debian $(shell uname -o) installer"
 # Inlude the build date of the installer in here.
 LSB_DISTRIB_RELEASE="$(DEBIAN_VERSION) - installer build $(BUILD_DATE)"
diff --git a/build/config/i386.cfg b/build/config/i386.cfg
index ba2d2db..054f3ca 100644
--- a/build/config/i386.cfg
+++ b/build/config/i386.cfg
@@ -2,7 +2,7 @@ MEDIUM_SUPPORTED = cdrom cdrom-xen netboot netboot-gtk netboot-xen hd-media
 MEDIUM_SUPPORTED_EXTRA = monolithic
 
 # The version of the kernel to use.
-BASEVERSION = 3.2.0-3
+BASEVERSION = 3.5.1-1
 KERNELVERSION = $(BASEVERSION)-486
 KERNELMAJOR = 2.6
 KERNEL_FLAVOUR = di
diff --git a/build/config/i386/cdrom.cfg b/build/config/i386/cdrom.cfg
index 5678ba5..ec36871 100644
--- a/build/config/i386/cdrom.cfg
+++ b/build/config/i386/cdrom.cfg
@@ -1,5 +1,5 @@
 # el-torito is too large at the moment, so is disabled.
-FLAVOUR_SUPPORTED = isolinux gtk #el-torito
+FLAVOUR_SUPPORTED = isolinux #gtk el-torito
 
 MEDIA_TYPE = CD-ROM
 
diff --git a/build/config/i386/hd-media.cfg b/build/config/i386/hd-media.cfg
index 4aa3977..38eee9d 100644
--- a/build/config/i386/hd-media.cfg
+++ b/build/config/i386/hd-media.cfg
@@ -1,4 +1,4 @@
-FLAVOUR_SUPPORTED = "" gtk
+FLAVOUR_SUPPORTED = ""
 
 # Not really a floppy, this is a 956 MiB image, large enough to put a
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
@@ -13,8 +13,8 @@ GZIPPED = .gz
 EXTRANAME = hd-media/
 
 # Add the gtk images on to the bootable image.
-EXTRATARGETS = build_hd-media_gtk
-INITRD_GTK = dest/hd-media/gtk/initrd.gz
+#EXTRATARGETS = build_hd-media_gtk
+#INITRD_GTK = dest/hd-media/gtk/initrd.gz
 TARGET = $(KERNEL) $(INITRD) $(BOOT)
 
 MANIFEST-BOOT = "1 gb image (compressed) for USB memory stick"
diff --git a/build/config/ia64.cfg b/build/config/ia64.cfg
index 5f7bb25..b085d5c 100644
--- a/build/config/ia64.cfg
+++ b/build/config/ia64.cfg
@@ -5,7 +5,7 @@ MEDIUM_SUPPORTED = cdrom netboot
 
 # The version of the kernel to use.
 KERNELMAJOR = 2.6
-KERNELVERSION = 3.2.0-3-itanium
+KERNELVERSION = 3.5.1-1-itanium
 KERNEL_FLAVOUR = di
 KERNELNAME = vmlinuz
 KERNELIMAGEVERSION = $(KERNELVERSION)
@@ -17,7 +17,7 @@ INITRD_FS = initramfs
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 
 arch_tree:
diff --git a/build/config/x86.cfg b/build/config/x86.cfg
index ece837f..eb75a83 100644
--- a/build/config/x86.cfg
+++ b/build/config/x86.cfg
@@ -8,7 +8,7 @@
 DOS_VOLUME_ID = deb00001
 # The DOS volume label to use for DOS floppies. This is a 11 character
 # string.
-DOS_VOLUME_LABEL = "Debian Inst"
+DOS_VOLUME_LABEL = "SprezzOS"
 
 # The image to use for a syslinux splash screen.
 SPLASH_PNG=boot/x86/pics/joy.png
@@ -24,7 +24,7 @@ XENCFG = $(SOME_DEST)/$(EXTRANAME)debian.cfg
 x86_syslinux:
 	-rm -rf $(TEMP_SYSLINUX)/*
 	mkdir -p $(TEMP_SYSLINUX)
-	TYPE=$(SYSLINUX_CFG) INCLUDE_GTK=$(INITRD_GTK) DESKTOP=$(DESKTOP) \
+	TYPE=$(SYSLINUX_CFG) DESKTOP=$(DESKTOP) \
 		syslinux-cfgs boot/x86 $(TEMP_SYSLINUX)
 
 # Compress binaries to save more space.
diff --git a/build/pkg-lists/cdrom-apus/common b/build/pkg-lists/cdrom-apus/common
index 69e5767..ac59d8d 100644
--- a/build/pkg-lists/cdrom-apus/common
+++ b/build/pkg-lists/cdrom-apus/common
@@ -14,8 +14,6 @@ di-utils-terminfo
 cdebconf-newt-udeb
 cdebconf-newt-terminal ?
 
-nano-udeb
-
 # This is so additional modules (eg, scsi) can be loaded from driver
 # floppies. cdrom-detect will use it.
 media-retriever
diff --git a/build/pkg-lists/cdrom/amd64.cfg b/build/pkg-lists/cdrom/amd64.cfg
index 91420aa..a4d2395 100644
--- a/build/pkg-lists/cdrom/amd64.cfg
+++ b/build/pkg-lists/cdrom/amd64.cfg
@@ -3,7 +3,6 @@ floppy-modules-${kernel:Version}
 fat-modules-${kernel:Version}
 cdrom-core-modules-${kernel:Version}
 input-modules-${kernel:Version}
-fb-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 usb-modules-${kernel:Version}
diff --git a/build/pkg-lists/cdrom/common b/build/pkg-lists/cdrom/common
index ca63580..3268cd7 100644
--- a/build/pkg-lists/cdrom/common
+++ b/build/pkg-lists/cdrom/common
@@ -19,8 +19,6 @@ preseed-common
 initrd-preseed
 file-preseed
 
-nano-udeb
-
 # This is so additional modules (eg, scsi) can be loaded from driver
 # floppies. cdrom-detect will use it.
 media-retriever
diff --git a/build/pkg-lists/cdrom/i386.cfg b/build/pkg-lists/cdrom/i386.cfg
index 5a77de6..8359e68 100644
--- a/build/pkg-lists/cdrom/i386.cfg
+++ b/build/pkg-lists/cdrom/i386.cfg
@@ -3,7 +3,6 @@ floppy-modules-${kernel:Version}
 fat-modules-${kernel:Version}
 cdrom-core-modules-${kernel:Version}
 input-modules-${kernel:Version}
-fb-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 usb-modules-${kernel:Version}
diff --git a/build/pkg-lists/floppy/common b/build/pkg-lists/floppy/common
index cfc0197..77ae3b6 100644
--- a/build/pkg-lists/floppy/common
+++ b/build/pkg-lists/floppy/common
@@ -19,8 +19,6 @@ kbd-udeb
 cdebconf-newt-udeb
 installation-locale
 
-nano-udeb
-
 # floppy loading support
 load-media
 media-retriever
diff --git a/build/pkg-lists/generic/common b/build/pkg-lists/generic/common
index fceddde..668dd52 100644
--- a/build/pkg-lists/generic/common
+++ b/build/pkg-lists/generic/common
@@ -11,8 +11,6 @@ cdebconf-newt-terminal ?
 installation-locale
 di-utils-terminfo
 
-nano-udeb
-
 preseed-common
 initrd-preseed
 network-preseed
diff --git a/build/pkg-lists/gtk-common b/build/pkg-lists/gtk-common
index 6f22ca6..48bb61e 100644
--- a/build/pkg-lists/gtk-common
+++ b/build/pkg-lists/gtk-common
@@ -4,23 +4,7 @@ rootskel-gtk
 cdebconf-gtk-udeb
 cdebconf-gtk-terminal
 
-xserver-xorg-core-udeb
 gtk2-engines-udeb
 
 # Fonts: see http://wiki.debian.org/DebianInstaller/GUIFonts
 ttf-freefont-udeb
-fonts-farsiweb-udeb
-ttf-cjk-compact-udeb
-fonts-tibetan-machine-udeb
-ttf-dejavu-udeb
-fonts-khmeros-udeb
-fonts-thai-tlwg-udeb
-fonts-taml-udeb
-fonts-mlym-udeb
-ttf-kannada-fonts-udeb
-fonts-telu-udeb
-fonts-sil-abyssinica-udeb
-fonts-lklug-sinhala-udeb
-fonts-lao-udeb
-fonts-ukij-uyghur-udeb
-fonts-sil-padauk-udeb
diff --git a/build/pkg-lists/hd-media/amd64.cfg b/build/pkg-lists/hd-media/amd64.cfg
index 415d281..8e4212f 100644
--- a/build/pkg-lists/hd-media/amd64.cfg
+++ b/build/pkg-lists/hd-media/amd64.cfg
@@ -1,6 +1,5 @@
 console-setup-pc-ekmap
 input-modules-${kernel:Version}
-fb-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 
@@ -24,10 +23,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/common b/build/pkg-lists/hd-media/common
index 395511e..00ff31c 100644
--- a/build/pkg-lists/hd-media/common
+++ b/build/pkg-lists/hd-media/common
@@ -14,8 +14,6 @@ preseed-common
 initrd-preseed
 file-preseed
 
-nano-udeb
-
 # Starting from 2.6.25, the isofs module is in a separate udeb
 isofs-modules-${kernel:Version} ?
 
diff --git a/build/pkg-lists/hd-media/gtk/amd64.cfg b/build/pkg-lists/hd-media/gtk/amd64.cfg
index 82f7c84..016671c 100644
--- a/build/pkg-lists/hd-media/gtk/amd64.cfg
+++ b/build/pkg-lists/hd-media/gtk/amd64.cfg
@@ -1,8 +1,6 @@
 #include "gtk-linux"
 
 mouse-modules-${kernel:Version}
-xserver-xorg-input-evdev-udeb
-xserver-xorg-video-fbdev-udeb
 
 speakup-modules-${kernel:Version}
 sound-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/gtk/i386.cfg b/build/pkg-lists/hd-media/gtk/i386.cfg
index 82f7c84..016671c 100644
--- a/build/pkg-lists/hd-media/gtk/i386.cfg
+++ b/build/pkg-lists/hd-media/gtk/i386.cfg
@@ -1,8 +1,6 @@
 #include "gtk-linux"
 
 mouse-modules-${kernel:Version}
-xserver-xorg-input-evdev-udeb
-xserver-xorg-video-fbdev-udeb
 
 speakup-modules-${kernel:Version}
 sound-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/i386.cfg b/build/pkg-lists/hd-media/i386.cfg
index 415d281..8e4212f 100644
--- a/build/pkg-lists/hd-media/i386.cfg
+++ b/build/pkg-lists/hd-media/i386.cfg
@@ -1,6 +1,5 @@
 console-setup-pc-ekmap
 input-modules-${kernel:Version}
-fb-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 
@@ -24,10 +23,6 @@ cdrom-core-modules-${kernel:Version}
 
 # all the filesystems (feel free to add more!)
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/hd-media/powerpc.cfg b/build/pkg-lists/hd-media/powerpc.cfg
index 97faa81..0355258 100644
--- a/build/pkg-lists/hd-media/powerpc.cfg
+++ b/build/pkg-lists/hd-media/powerpc.cfg
@@ -19,10 +19,6 @@ sata-modules-${kernel:Version}
 # all the filesystems (feel free to add more!)
 hfs-modules-${kernel:Version}
 fat-modules-${kernel:Version}
-ext2-modules-${kernel:Version}
-ext3-modules-${kernel:Version}
-ext4-modules-${kernel:Version}
-reiserfs-modules-${kernel:Version}
 
 # the iso is loop-mounted
 loop-modules-${kernel:Version}
diff --git a/build/pkg-lists/nativehd/common b/build/pkg-lists/nativehd/common
index 0a76ebd..23d781b 100644
--- a/build/pkg-lists/nativehd/common
+++ b/build/pkg-lists/nativehd/common
@@ -18,8 +18,6 @@ hw-detect
 ethdetect
 netcfg
 
-nano-udeb
-
 libfribidi0-udeb
 
 preseed-common
diff --git a/build/pkg-lists/netboot-apus/common b/build/pkg-lists/netboot-apus/common
index 16c0db0..80863b1 100644
--- a/build/pkg-lists/netboot-apus/common
+++ b/build/pkg-lists/netboot-apus/common
@@ -11,6 +11,4 @@ cdebconf-newt-terminal ?
 installation-locale
 di-utils-terminfo
 
-nano-udeb
-
 libfribidi0-udeb
diff --git a/build/pkg-lists/netboot/amd64.cfg b/build/pkg-lists/netboot/amd64.cfg
index ec6beef..ef72adc 100644
--- a/build/pkg-lists/netboot/amd64.cfg
+++ b/build/pkg-lists/netboot/amd64.cfg
@@ -12,9 +12,6 @@ input-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 
-# This is needed for proper display of utf-8.
-fb-modules-${kernel:Version}
-
 # In case they need to load a driver image.
 mountmedia
 media-retriever
diff --git a/build/pkg-lists/netboot/common b/build/pkg-lists/netboot/common
index 823cdc5..07ca0cf 100644
--- a/build/pkg-lists/netboot/common
+++ b/build/pkg-lists/netboot/common
@@ -16,8 +16,6 @@ initrd-preseed
 network-preseed
 file-preseed
 
-nano-udeb
-
 save-logs
 
 libfribidi0-udeb
diff --git a/build/pkg-lists/netboot/i386.cfg b/build/pkg-lists/netboot/i386.cfg
index 3340b1d..0f11356 100644
--- a/build/pkg-lists/netboot/i386.cfg
+++ b/build/pkg-lists/netboot/i386.cfg
@@ -12,9 +12,6 @@ input-modules-${kernel:Version}
 console-setup-udeb
 kbd-udeb
 
-# This is needed for proper display of utf-8.
-fb-modules-${kernel:Version}
-
 # In case they need to load a driver image.
 mountmedia
 media-retriever
